src/constants/actions.js

```diff
@@ -5,14 +5,14 @@ export const ACTIONS = {
     field: "week_stargazers_count",
     order: "DESC",
   },
-  MOST_POPULAR: {
-    label: "Most popular",
-    route: "/most-popular",
+  POPULAR: {
```

Comment:
Hey @roya3000 ! Thanks for taking this up.

I actually only meant to change to route URL, and not the labels, but now I am liking this change more and more.

I wonder if all the labels should change, then. Because now, `newest` doesn't follow the same naming as `Popular`, if you get what I mean.

Something like that would be more consistent:

<img width="495" alt="Screen Shot 2020-10-01 at 08 28 29" src="https://user-images.githubusercontent.com/5920450/94809096-18d2bb80-03c0-11eb-873b-08b1aee5b086.png">

I just wish there was a shorter way of saying "Recently updated" 😅

Feedback on all of this?

Thanks again!

Reply:
Hi @reobin,

Thanks for checking this PR.

I agree, the changes for Popular -> Top, Newes -> New, Oldest -> Old make total sense. (I will amend the changes in this PR)

I think "Recently updated" is a clear way to say, that theme was updated, possible replacement could be "New release", however we will confuse users, because its not clear difference between "New releases" and "New"

Reply:
I like it! Thanks for your work @roya3000